ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/FakeInsertValuesExecutor.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.test.tools;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.confluent.ksql.KsqlExecutionContext;
+import io.confluent.ksql.engine.InsertValuesEngine;
+import io.confluent.ksql.parser.tree.InsertValues;
+import io.confluent.ksql.services.ServiceContext;
+import io.confluent.ksql.test.serde.string.StringSerdeSupplier;
+import io.confluent.ksql.test.tools.exceptions.InvalidFieldException;
+import io.confluent.ksql.util.KsqlConfig;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+
+import org.apache.kafka.clients.producer.ProducerRecord;
+
+public class FakeInsertValuesExecutor extends InsertValuesEngine {
+  private final FakeKafkaService fakeKafkaService;
+
+  public FakeInsertValuesExecutor(FakeKafkaService fakeKafkaService) {
+    super(() -> 0);
```

Comment:
Hmm, I think we should send the current system time here.
You can change it to:
`super(System::currentTimeMillis)`

Reply:
Or actually you can just call `super()`.

Reply:
I left it at 0 because the default timestamp for inputs from the `input.json` file that don't have a specified timestamp is also 0. Should I change both?

Reply:
This should behave the same as the `INSERT INTO VALUES` in KSQL. Over there we use system current time so let's use the same. User can specify the timestamp if they desire.

Reply:
@hjafarpour - I disagree here, using real timestamps in tests can cause flakiness. I think I prefer always having timestamp 0.

Reply:
You can set the timestamp column in the statement if you need to.

Reply:
Because writing the correct timestamp in the outputs file is impossible, having no specified expected timestamp will skip checking for it if it was run with no input file.

Reply:
Then make sure that you have this documented. Default ROWTIME value of zero will have implications for queries such as windowed aggregations.